src/Avalonia.DotNetCoreRuntime/NetCoreRuntimePlatform.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Avalonia.Shared.PlatformSupport
+{
+    internal partial class StandardRuntimePlatform
```

Comment:
I'm wondering why this is a partial class implemented in two files? Also neither file has the name of the class that's contained in it, which is a little confusing. Should `NetCoreRuntimePlatform.cs` be named `StandardRuntimePlatform.cs`?